mne/decoding/tests/test_time_gen.py

```diff
@@ -424,4 +424,23 @@ def test_decoding_time():
     tg = TimeDecoding(clf=clf, cv=cv)
     tg.fit(epochs, y=y)
 
+
+def test_string_scorer():
+    """Test whether passing string as scorer works as intended"""
+    epochs = make_epochs()
+    gat_1 = GeneralizationAcrossTime(scorer='accuracy')
+    gat_1.fit(epochs)
+    assert_equal("<GAT | fitted, start : -0.100 (s), stop : 0.500 (s), "
+                 "no prediction, no score>", "%s" %gat)
+    predict_1 = gat_1.predict(epochs)
+    score_1 = gat_1.score()
```

Comment:
The predict line isn't necessary. You can directly do `gat_1.score(epochs)`. Same below.